### ISSUE TITLE
Pre-trained checkpoints

### ISSUE THREAD

**Issue by MinamiKotoka** [Jingkang50/PSG4D#3]
Hi, do you plan to release the pre-trained seg/track checkpoints on sailvos3d and hoi4d datasets? Thanks!

**Comment by nguyentthong** [comment]
Hi @Jingkang50  @Jun-CEN  @LilyDaytoy  @choiszt, I have the same question as well. I have trained the rgbd segmentation model on the sailvos3d dataset. Particularly, these are the configs I use:

- For training: configs/mask2former/mask2former_dual_rgbd.py
- For testing: psg_4d_unitrack.py
- Epochs: 12
- Learning rate: 1e-4

Unfortunately, the results have not been so reasonable. In particular, these are some points with the segmentation model:

- The model only works effectively with the person and car object.
- It usually mis-detects other objects such as table, chair, and potted plant.

<img width="583" alt="Screenshot 2024-06-16 at 11 23 08" src="https://github.com/Jingkang50/PSG4D/assets/12288136/7e60ce99-c9e2-4833-9be7-86a8ac6b0852">
<img width="575" alt="Screenshot 2024-06-16 at 11 21 58" src="https://github.com/Jingkang50/PSG4D/assets/12288136/be520b14-13df-4077-8a6e-ba11cb09153a">

As such, can you release segmentation checkpoints to facilitate reproducibility? Thank you so much.
